server/api/global_secret.go

```diff
@@ -31,11 +32,15 @@ func GetGlobalSecretList(c *gin.Context) {
 		c.String(http.StatusInternalServerError, "Error getting global secret list. %s", err)
 		return
 	}
-	// copy the secret detail to remove the sensitive
-	// password and token fields.
-	for i, secret := range list {
-		list[i] = secret.Copy()
+
+	if os.Getenv("WOODPECKER_SECRET_ALLOW_SHOW_VALUE") != "true" {
```

Comment:
this should be a config value stored in server config and not read each time from env-var